docs/pages/functionality/patterns.md

```diff
@@ -138,4 +138,4 @@ Deployed. Endpoint(s) available at: http://localhost:8000/catalog
 
 ## Related Reading
 
-- [`mesheryctl pattern`](/reference/mesheryctl/#service-mesh-pattern-configuration-and-management)
+- [`mesheryctl pattern`](https://docs.meshery.io/reference/mesheryctl/pattern)
```

Comment:
```suggestion
- [`mesheryctl pattern`]({{ site.baseurl }}/reference/mesheryctl/pattern)
```

Reply:
Let's be sure to use the Jekyll / liquid tag in order to avoid hardcoding the site's URL.